source/frontend/src/features/leases/detail/LeasePages/tenant/TenantOrganizationContactInfo.tsx

```diff
@@ -34,25 +34,25 @@ export const TenantOrganizationContactInfo: React.FunctionComponent<
   const primaryContactName = formatApiPersonNames(primaryContact);
   return (
     <StyledSectionWrapper>
-      <SectionField labelWidth="2" contentWidth="4" label="Organization">
```

Comment:
sections weirdly set to less then half the screen, think this may have been copy/pasted from contacts where a half screen design is being used.